File: we_4_7_drift_across_grades.m
% Integrated Navigation Systems — Chapter 4: Inertial Sensors and Error Modeling
% Worked Example 4.7 — Drift across grades
% Companion code (educational reference implementation; see the book for context).

g=9.81; t=100;
for bhr=[1 0.01]
  bg=deg2rad(bhr)/3600; dp=g*bg*t^3/6;
  fprintf('%.3f deg/hr -> %.3f m\n',bhr,dp);
end   % 7.927, 0.079

File: we_4_7_drift_across_grades.py
# Integrated Navigation Systems — Chapter 4: Inertial Sensors and Error Modeling
# Worked Example 4.7 — Drift across grades
# Companion code (educational reference implementation; see the book for context).

import numpy as np
g=9.81; t=100
for bhr in (1,0.01):
    bg=np.radians(bhr)/3600; print(f'{bhr} deg/hr -> {g*bg*t**3/6:.3f} m')  # 7.927, 0.079
